CHANGELOG.md

```diff
@@ -6,12 +6,19 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- `PodListeners` CRD ([#644]).
+
+### Changed
+
+- Derive `Eq` and `Copy` where applicable for listener CRDs ([#644]).
+
 - Add support for tls pkcs12 password to secret operator volume builder ([#645]).
```

Comment:
I think the merged screwed this up (should be under added)

Reply:
fixed in c6cae866ed9121b04183c69512f171fdd5ca4d76